ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -23,7 +23,7 @@ resources:
       memory: 200Mi
 tests:
 - as: control-plane-120nodes
-  cron: 0 16 11,25 * *
+  cron: 0 16 18 * *
```

Comment:
suggest: `0 16 11 * *` or day=25

We will have uneven number of jobs on day=18 with this expression.